src/hashTable.js

```diff
@@ -0,0 +1,64 @@
+export default class HashTable {
+  constructor() {
+    this.hashTable = []
+    this.count = 0
+  }
+
+  hash( key ) {
+    let sum = 0;
+    const randomH = 31
+    const prime = 51
+    key = key.toString();
+    for (let i = 0; i < key.length; i++) {
+        sum = (randomH * sum) + key.charCodeAt(i)
+    }
+    return sum % prime // return hashingIndex
+  }
+
+  put( key, value ) {
+    const hashedKey = this.hash( key )
+    this.hashTable[hashedKey] = { 'key': key, 'value':value }
+    this.count++
+  }
+
+  get( key ) {
+    const hashedKey = this.hash( key )
+    const value = this.hashTable[hashedKey].value
+    if( value == undefined ) {
```

Comment:
Best practice is to use threequals `===` so that you are comparing type and value (though I don't think it's strictly necessary for `undefined`